Bottom: 3b4dda0b1692755f9a28b8a7f3943e3b8712ea4a
Top:    d368ffa61d4f438c63152f377eae210092a99150
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:34:06 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/select.c b/src/select.c
index 32161ec..3074075 100644
--- a/src/select.c
+++ b/src/select.c
@@ -961,12 +961,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
     {
       SaveFindFlag (FOUNDFLAG);
       InitConnectionLookup ();
-      //changed = ResetConnections (true) || changed;
-      {
-        extern int TheFlag;
-        printf ("select.c: THE FLAG IS %i\n", TheFlag);
-        changed = ResetConnections (true, TheFlag) || changed;
-      }
+      changed = ResetConnections (true, FOUNDFLAG) || changed;
 
       MENU_LOOP (&PCB->NetlistLib);
       {
@@ -987,12 +982,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       END_LOOP;
 
       changed = SelectConnection (Flag) || changed;
-      //changed = ResetConnections (false, TheFlag) || changed;
-      {
-        extern int TheFlag;
-        printf ("select.c: THE FLAG IS %i\n", TheFlag);
-        changed = ResetConnections (false, TheFlag) || changed;
-      }
+      changed = ResetConnections (false, FOUNDFLAG) || changed;
       FreeConnectionLookupMemory ();
       RestoreFindFlag ();
     }
